lib/encoder.py

```diff
@@ -836,29 +857,33 @@ def StoreEncoding(self, encoding, workdir=None):
   def ReadEncodingResult(self, encoding, scoredir=None):
     """Reads an encoding result back from storage, if present.
 
-    None is returned file does not exist."""
+    None is returned if file does not exist.
+    If scoredir is given, only that directory is searched.
```

Comment:
I have deliberately left off the parameter documentation in all cases so far. I know it's useful (especially with Python's weak typing system), but I want to either do it or not do it.